modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -29,20 +29,21 @@ spec:
         devices:
           interfaces:
             - masquerade: {}
-              name: default
+              name: default <1>
             - bridge: {}
-              name: bridge-net <1>
+              name: bridge-net <2>
 # ...
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <2>
+        - name: bridge-net <3>
           multus:
-            networkName: a-bridge-network <3>
+            networkName: a-bridge-network <4>
 ----
-<1> The name of the bridge interface.
-<2> The name of the network. This value must match the `name` value of the corresponding `spec.template.spec.domain.devices.interfaces` entry.
-<3> The name of the network attachment definition.
+<1> Optional: The name of the primary default pod network.
```

Comment:
Some people may understand this as - the `name` field is optional. While the whole items in the `interfaces` and `networks` lists is optional.

Vast majority of the customers attach to secondary networks only. I would suggest removing the default network from our "this is how you connect to a secondary network" examples altogether.

We could compensate for that removal in the network overview introduction:

> VMs can be connected to the default pod network and to any number of secondary networks.

Not sure about the exact wording, but it should explain that there can be 0/1 connections to the pod network and 0-N to secondary networks.

---

Alternatively we can adjust these callouts to refer to whole "default" items in `interfaces` and `networks`  sections.